package.json

```diff
@@ -8,7 +8,9 @@
     "@testing-library/user-event": "^12.1.10",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-redux": "^7.2.4",
     "react-scripts": "4.0.3",
+    "redux": "^4.1.1",
     "web-vitals": "^1.0.1"
   },
```

Comment:
- [ ] <b>REQUIRED:</b> `prop-types` is missing from dependencies. Kindly add it to package.json for [typechecking props](https://reactjs.org/docs/typechecking-with-proptypes.html) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/section/propTypes.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896806/))